src/t8_eclass.c

```diff
@@ -159,6 +159,13 @@ t8_eclass_compare (t8_eclass_t eclass1, t8_eclass_t eclass2)
   }
 }
 
+int
+t8_eclass_refines_irregular (t8_eclass_t eclass)
+{
+  /*Pyramids are currently the only irregular eclass */
+  return eclass == T8_ECLASS_PYRAMID;
+}
+
```

Comment:
The refinement pattern is independent of the element class.
eclass is a concept for coarse mesh elements and not for fine elements.

This function should be part of the `t8_eclass_scheme` interface instead.

Reply:
Done